src/views/components/avatarHome/avatarHome.tsx

```diff
@@ -228,11 +217,17 @@ export function AvatarHome() {
         </h4>
       </div>
 
+      <h4>
+        <strong style={{ color: '#1976D2' }}>
+          Until the DD/MM/YYYY, all users will be given a minimum avatar library
+          size of 2!
+        </strong>
+      </h4>
```

Comment:
Let's add the patreon stuff here in like 4 dot points or something super simple.